README.md

```diff
@@ -13,15 +13,22 @@ SYNOPSIS
 DESCRIPTION
 -----------
 
-A command line tool that creates a pdf document containing the license for a given dataset. The tool searches for a dataset that corresponds to the given `datasetID` and uses the metadata of this dataset, as well as the personal data of the depositor to generate the license.
+A command line tool that creates a pdf document containing the license for a given dataset. The tool searches for a dataset that corresponds 
```

Comment:
Some minor editing, mainly making the lines a bit shorter to make the text readible in a plain text editor.